src/androidTest/java/pl/edu/zut/mad/schedule/ScheduleFragmentTest.kt

```diff
@@ -0,0 +1,114 @@
+package pl.edu.zut.mad.schedule
+
+import android.app.Activity.RESULT_CANCELED
+import android.app.Activity.RESULT_OK
+import android.support.test.espresso.Espresso
+import android.support.test.espresso.Espresso.onView
+import android.support.test.espresso.assertion.ViewAssertions.matches
+import android.support.test.espresso.intent.Intents
+import android.support.test.espresso.intent.Intents.intended
+import android.support.test.espresso.intent.matcher.IntentMatchers.hasComponent
+import android.support.test.espresso.matcher.ViewMatchers.Visibility.VISIBLE
+import android.support.test.espresso.matcher.ViewMatchers.withEffectiveVisibility
+import android.support.test.espresso.matcher.ViewMatchers.withText
+import android.support.test.rule.ActivityTestRule
+import android.support.v7.app.AppCompatActivity
+import com.nhaarman.mockito_kotlin.mock
+import com.nhaarman.mockito_kotlin.times
+import com.nhaarman.mockito_kotlin.verify
+import junit.framework.Assert.assertTrue
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import pl.edu.zut.mad.schedule.login.LoginActivity
+import pl.edu.zut.mad.schedule.module.ScheduleComponent
+
+internal class ScheduleFragmentTest {
+
+    @get:Rule
+    val activityRule = ActivityTestRule(AppCompatActivity::class.java)
+    val scheduleFragment = TestScheduleFragment()
+
+    val fragmentStartsIdlingResource by lazy {
+        FragmentStartsIdlingResource(activityRule.activity, TestScheduleFragment::class)
+    }
+
+    @Before
+    fun setUp() {
+        Intents.init()
+        startFragment()
+        Espresso.registerIdlingResources(fragmentStartsIdlingResource)
+    }
+
+    @Test
+    fun presenterOnViewIsCreatedShouldBeCalledWhenActivityResultIsOk() {
+        scheduleFragment.onActivityResult(ScheduleFragment.REQUEST_CODE, RESULT_OK, null)
+
+        verify(scheduleFragment.presenter, times(2)).onViewIsCreated()
```

Comment:
I know that `times(2)` should be omitted but `onViewIsCreated` is called also from `Fragment.onViewIsCreated()`

Reply:
You could just name it somehow to describe what you wrote here :)